### ISSUE TITLE
Backstage should allow inspecting k8s resources from workload clusters

### ISSUE THREAD

**Issue by marians** [giantswarm/roadmap#3297]
Currently the Kubernetes plugin in our Dev Portal only discovers annotated resources in management clusters.

To extend the visibility of Kubernetes resources also for applications deployed to workload clusters, we should include workload clusters here.

There are some technical prerequisites to be figured out:

- Dynamic configuration/discovery of clusters
- Authorization for Backstage to look up resources in workload clusters e. g. via a service account

**Comment by weatherhog** [comment]
@gusevda this issue has been in watching for the last 4 months, can this be closed?

**Comment by stone-z** [comment]
We haven't done more with pinniped since the initial work by BigMac. I wasn't aware anyone was waiting for something 🙈

**Comment by weatherhog** [comment]
@stone-z yes, we would still have a deeper look into pinniped and therefor a investigation from you or @giantswarm/team-shield would be great.

**Comment by marians** [comment]
**A little status update:** Last year we looked into Pinniped. it was working, but would result in a UX where users would have to authenticate with even more clusters individually. As a result, we started re-evaluating our SSO model, which currently requires a separate auth token per management cluster.
